power_grid_model_c/power_grid_model_c/power_grid_model_c.cpp

```diff
@@ -300,7 +300,7 @@ void PGM_update_model(PGM_Handle* handle, PGM_PowerGridModel* model, PGM_Idx n_c
         dataset[components[i]] = ConstDataPointer{update_data[i], component_sizes[i]};
     }
     try {
-        model->update_component(dataset);
+        model->update_component<MainModel::permanent_update_t>(dataset);
```

Comment:
TODO: make this `cached_update_t`

Reply:
This call is intended to do permanent update.